stdlib/Markdown/docs/src/index.md

```diff
@@ -67,6 +69,12 @@ A paragraph containing some ``\LaTeX`` markup.
     ```
     @doc raw"``\LaTeX`` syntax in a docstring." functionname
     ```
+    
+!!! note
+    Inline ``\LaTeX`` may also be created within a set of single of `$` characters, though this is 
+    depreciated. The ``` `` ``` syntax is preferred as `$` is also used for string interpolation -
+    the parser will interpret an accidentally unmatched `$` as an intented string interpolation 
+    whereas it will alert the user to an unmatched ``` `` ```.
```

Comment:
> it will alert the user to an unmatched

I don't think this is true? Any sequence of characters should get parsed as Markdown without errors.